spec/visual_report_helper.rb

```diff
@@ -1,6 +1,46 @@
 # frozen_string_literal: true
 
 module VisualReportHelper
+  VISUAL_REPORT_TEMPLATE = <<-HTML
```

Comment:
Feels more like a loophole than anything, and an oversight on Rubocops part to not have a style governing constant length. :wink:

Reply:
I think about it more like "it's a bad habit to put long strings inside business logic, as it makes the code hard to read, but a constant is reasonably isolated and therefore acceptable". Plus there's still a protection given by [Metrics/ModuleLength](https://docs.rubocop.org/rubocop/cops_metrics.html#metricsmodulelength).

I'm merging assuming that we both agree that this change makes things better, but if you feel strongly about it we can always extract this template to a separate file in a dedicated PR :+1:

Reply:
No strong feelings, just an observation. :stuck_out_tongue: . It does make it easier to read, but it's still surprising it (Rubocop) doesn't have an opinion on this sort of thing too.